homeassistant/components/yolink/cover.py

```diff
@@ -27,25 +27,24 @@ async def async_setup_entry(
     entities = [
         YoLinkCoverEntity(config_entry, device_coordinator)
         for device_coordinator in device_coordinators.values()
-        if device_coordinator.device.device_type == ATTR_DEVICE_DOOR_SENSOR
-        and device_coordinator.device.parent_id is not None
-        and device_coordinator.device.parent_id != "null"
+        if device_coordinator.device.device_type == ATTR_GARAGE_DOOR_CONTROLLER
     ]
     async_add_entities(entities)
 
 
 class YoLinkCoverEntity(YoLinkEntity, CoverEntity):
     """YoLink Cover Entity."""
 
+    _attr_has_entity_name = True
+
     def __init__(
         self,
         config_entry: ConfigEntry,
         coordinator: YoLinkCoordinator,
     ) -> None:
         """Init YoLink garage door entity."""
         super().__init__(config_entry, coordinator)
-        self._attr_unique_id = f"{coordinator.device.device_id}_door_state"
-        self._attr_name = f"{coordinator.device.device_name} (State)"
+        self._attr_unique_id = f"{coordinator.device.device_id}"
```

Comment:
Changing the entity unique_id is not allowed without a migration. Do we really need to change it?

Reply:
No, I will rollback this change